packages/salesforcedx-vscode-soql/src/editor/soqlEditorInstance.ts

```diff
@@ -181,11 +182,23 @@ export class SOQLEditorInstance {
         break;
       }
       case MessageType.RUN_SOQL_QUERY: {
-        this.handleRunQuery().catch(() => {
-          channelService.appendLine(
-            `An error occurred while running the SOQL query.`
-          );
-        });
+        vscode.window
+          .withProgress(
+            {
+              cancellable: false,
+              location: vscode.ProgressLocation.Notification,
+              title: 'Running query'
```

Comment:
Add this to `src/messages/i18n.ts`

cc @ekapner

Reply:
@jgellin-sf - I do not have this one in my other PR.  https://github.com/forcedotcom/salesforcedx-vscode/pull/2810

Reply:
Thanks. I covered it now in the `chore(soql): i18n for new message` commit.